ipn/ipnlocal/local.go

```diff
@@ -4848,6 +4849,40 @@ func (b *LocalBackend) CheckIPForwarding() error {
 	return warn
 }
 
+func (b *LocalBackend) CheckUDPGROForwarding() error {
```

Comment:
exported things need docs

Reply:
Updated in https://github.com/tailscale/tailscale/pull/10071/commits/80713c8ec24687db8d51bd21b9a3df5849483daf